README.md

```diff
@@ -20,40 +20,27 @@ EPD | Controller
 --- | ---
 VCC | 3V3
 GND | GND
-SDI | MOSI
-SCLK | SCK
+SDI | MOSI(ESP32: 23)
+SCLK | SCK(ESP32: 18)
 CS | 10
 DC | 9
 Reset | 8
 Busy | 7
 
-### ESP32
 You can change pin assign for CS, DC, Reset and Busy like this.
 ```
 EPD1in54 epd(33, 25, 26, 27); // reset, dc, cs, busy
 // EPD1in54B epd(33, 25, 26, 27); // for red
 // EPD1in54C epd(33, 25, 26, 27); // for yellow
 ```
 
-EPD | Controller
---- | ---
-VCC | 3V3
-GND | GND
-SDI | MOSI(ESP32: 23)
-SCLK | SCK(ESP32: 18)
-CS | 26
-DC | 25
-Reset | 33
-Busy | 27
-
 ## Program
 See [examples](./examples).
 
 # License
 MIT
 
 # References
-- [ESP32で1.54インチのePaperを動かす方法](http://asukiaaa.blogspot.com/2019/01/esp32154epaper.html)
```

Comment:
Why did you remove this link?
For ESP32-WROOM-32 `GPIO6 to GPIO11 are connectedto the integrated SPI flash integrated on the module and are not recommended for other uses.`
https://www.espressif.com/sites/default/files/documentation/esp32-wroom-32_datasheet_en.pdf

Reply:
I'm sorry, I lost the link unintentionally